source/localizable/testing/testing-components.md

```diff
@@ -109,13 +109,12 @@ export default Ember.Component.extend({
 ```app/templates/components/magic-title.hbs
 <h2>{{title}}</h2>
 
-<button {{action "updateTitle"}}>
+<button class="title-button" {{action "updateTitle"}}>
   Update Title
 </button>
 ```
 
-jQuery triggers can be used to simulate user interaction and test that the title
-is updated when the button is clicked on:
+We recommend using native DOM events or the [`ember-native-dom-helpers`](https://github.com/cibernox/ember-native-dom-helpers) addon to simulate user interaction and test that the title is updated when the button is clicked on. Using jQuery to simulate user click events might lead to unexpected test results as the action can potentially be called twice.
```

Comment:
Also, we can update the end of the first sentence to be just `title is update when the button is clicked.` We can cut `on` since we shouldn't end a sentence with a preposition.